### PR TITLE
released whl package for different spark version

### PR DESCRIPTION
**Pull request by pinggao187**

### CODE REVIEW

**Comment by pinggao187** [comment]
jenkins configure
http://10.239.176.111:18888/view/ZOO-PR/job/ZOO-PR-Release-PyPI/configure

File: pyzoo/dev/release_default_linux.sh [hunk 1]
```diff
@@ -33,5 +33,8 @@ if (( $# < 1)); then
 fi
 
 version=$1
+spark_version=$2
+bigdl_artifactId=$3
+spark=$4
```
Comment (glorysdj) [inline pyzoo/dev/release_default_linux.sh:38]: spark_profile?

Comment (glorysdj) [reply pyzoo/dev/release_default_linux.sh:38]: and give a default value?

File: pyzoo/dev/release_default_linux.sh [hunk 2]
```diff
@@ -33,5 +33,8 @@ if (( $# < 1)); then
 fi
 
 version=$1
+spark_version=$2
+bigdl_artifactId=$3
+spark_profile=$4
 
-bash ${RUN_SCRIPT_DIR}/release.sh linux ${version} false true -Dspark.version=2.4.3 -Dbigdl.artifactId=bigdl-SPARK_2.4 -P spark_2.4+
+bash ${RUN_SCRIPT_DIR}/release.sh linux ${version} false false -Dspark.version=${spark_version} -Dbigdl.artifactId=${bigdl_artifactId} -P ${spark_profile}
```
Comment (hkvision) [inline pyzoo/dev/release_default_linux.sh:40]: why change true to false?

Comment (pinggao187) [reply pyzoo/dev/release_default_linux.sh:40]: It's a test now, whl did not upload

**Comment by pinggao187** [comment]
update jenkins
http://10.239.176.111:18888/view/ZOO-PR/job/ZOO-PR-Release-PyPI/configure

**Comment by hkvision** [comment]
rename to release_linux.sh?

File: pyzoo/dev/release_default_spark_linux.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# If you want to customize the release, please use release.sh and specify maven parameters instead.
+
+set -e
+RUN_SCRIPT_DIR=$(cd $(dirname $0) ; pwd)
+echo $RUN_SCRIPT_DIR
+
+if (( $# < 1)); then
+  echo "Usage: release_default_linux.sh version"
+  echo "Usage example: bash release_default_linux.sh default"
+  echo "Usage example: bash release_default_linux.sh 0.6.0.dev0"
```
Comment (hkvision) [inline pyzoo/dev/release_default_spark_linux.sh:31]: modify these

File: pyzoo/dev/release_default_spark_linux.sh [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  exit -1
+fi
+
+version=$1
+spark_version=$2
+bigdl_artifactId=$3
+spark_profile=$4
+
+bash ${RUN_SCRIPT_DIR}/release.sh linux ${version} false true  -Dspark.version=${spark_version} -Dbigdl.artifactId=${bigdl_artifactId} -P ${spark_profile}
```
Comment (hkvision) [inline pyzoo/dev/release_default_spark_linux.sh:40]: remove the redundant space before -Dspark.version

File: pyzoo/dev/release_linux.sh [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is the default script with maven parameters to release analytics-zoo for linux.
+# Note that if the maven parameters to build analytics-zoo need to be changed,
+# make sure to change this file accordingly.
+# If you want to customize the release, please use release.sh and specify maven parameters instead.
```
Comment (hkvision) [inline pyzoo/dev/release_linux.sh:22]: Change to 
```
This is the script to release analytics-zoo for linux with different spark versions.
```
Also change for mac.

File: pyzoo/dev/release_linux.sh [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+# This is the script to release analytics-zoo for linux with different spark versions.
+
+set -e
+RUN_SCRIPT_DIR=$(cd $(dirname $0) ; pwd)
+echo $RUN_SCRIPT_DIR
+
+if (( $# < 1)); then
+  echo "Usage: release_linux.sh version spark_version bigdl_artifactId spark_profile"
+  echo "Usage example: bash release_linux.sh default spark_version bigdl_artifactId spark_profile"
```
Comment (hkvision) [inline pyzoo/dev/release_linux.sh:27]: replace spark_version bigdl_artifactId spark_profile with something concrete like 2.4.3 ...?

**Review by hkvision — APPROVED** [review]

**Pull request merged**